### ISSUE TITLE
Extent seems to break with Timeseries charts

### ISSUE THREAD

**Issue by Holyphoenix** [naver/billboard.js#142]
## Extent seems to break with Timeseries charts  
I was using Billboardjs to try and create a timeline (I know it doesn't actually support that).  In the example I created, I succeeded in creating a timeline, but found that the subchart wasn't functional.  I tried removing various different objects I had added but the subchart never came back.  I copied the billboardjs subchart example and it worked just fine.  I also tried running my test code with c3, and the subchart worked just fine.  

Notice in the screenshot there is no data shown and in the subchart.

![image](https://user-images.githubusercontent.com/3467817/30564885-069078b6-9c8c-11e7-9bf1-87ad53f17680.png)

## Steps to check or reproduce
http://jsfiddle.net/JimmyDev/4qh88q6r/

**Comment by Holyphoenix** [comment]
Actually, after more digging it looks like my "extent" attribute is setup wrong for Billboardjs.  It looks like it is different between c3 and billboard.  I got it figured out.  Thanks!

**Comment by Holyphoenix** [comment]
I am re-opening because I think there may be an issue here.  When I update the array to set both the x and the y, I get javascript errors saying that it can't parse as a date.  I have a feeling the extent attribute is broken with dates?

**Comment by netil** [comment]
Thanks @Holyphoenix  for the report.

The one of the hardest part upgrading to support D3 v4 was about scaling and  I think this is one of the related issue.
I'll try look for, but it'll be treated on next release(not the upcoming 1.1.0 due to the schedule).

**Comment by doug-the-guy** [comment]
I know this is old, but in the process of updating to `billboardjs 2.2.3 / d3 v4` from `c3 v0.4 / d3 v3` I had to modify how I set the default subchart zoom. According to the [c3 ref](https://c3js.org/reference.html#axis-x-extent) this is how you set the default zoom, but that was generating errors like:

> d3 error g attribute transform expected number NaN etc...

Using `chart.zoom(start, end)` works well as a substitute. I recommend noting that in the API docs until extent is fixed or removed.
